### DISCUSSION TITLE
Unable to create a support case

### DISCUSSION THREAD

**Discussion by malt03** [vercel/community#6272] [Account Issue] [answered]
### Question

https://vercel.com/docs/dashboard-features/support-center
I am on the pro plan so the support center should be enabled, but the "Create Case" button is a link to https://vercel.com/help and I cannot create a case.

https://vercel.com/pricing
This page should allow me to get Email Support, but I can't find an email address for it.

I would like to know how to get support.

**Comment by swarnava** [comment, marked as answer]
Hey there, on https://vercel.com/help if you scroll down, you should be able to find `Get Help` option which you can use to `create case` . Let us know if you are still facing any difficulties .

**Comment by malt03** [comment]
Thanks. I tried that button, but it only allowed me to talk to the AI, but not to create a case.

**Comment by malt03** [comment]
Oops, I hit the Get Help button and scrolled further down and found a form to create a support case. Thanks.
